Userland/Libraries/LibHTTP/HttpRequest.cpp

```diff
@@ -62,17 +62,29 @@ ErrorOr<ByteBuffer> HttpRequest::to_raw_request() const
     if (m_url.port().has_value())
         TRY(builder.try_appendff(":{}", *m_url.port()));
     TRY(builder.try_append("\r\n"sv));
+    // start headers
```

Comment:
```suggestion
    // Start headers
```
etc. for the other new comments too.